dist/svg.select.js

```diff
@@ -167,20 +167,43 @@ SelectHandler.prototype.drawCircles = function () {
         })(i);
 
         // add every point to the set
-        this.pointSelection.set.add(
-            // a circle with our css-classes and a touchstart-event which fires our event for moving points
-            this.nested.circle(this.options.radius)
-                .center(array[i][0], array[i][1])
-                .addClass(this.options.classPoints)
-                .addClass(this.options.classPoints + '_point')
-                .on('touchstart', curriedEvent)
-                .on('mousedown', curriedEvent)
-        );
+        // add css-classes and a touchstart-event which fires our event for moving points
+        var point = this.drawPoint(array[i][0], array[i][1])
+                        .addClass(this.options.classPoints)
+                        .addClass(this.options.classPoints + '_point')
+                        .on('touchstart', curriedEvent)
+                        .on('mousedown', curriedEvent)
+        this.pointSelection.set.add(point);
     }
+};
+
+// The function to draw single point
+SelectHandler.prototype.drawPoint = function (cx, cy) {
+    var pointType = this.options.pointType;
+
+    switch (pointType) {
+        case 'circle':
+            return this.drawCircle(cx, cy);
+        case 'rect':
+            return this.drawRect(cx, cy);
+        default:
+            throw 'Unknown ' + pointType + ' point type!'
```

Comment:
Throwing a string is not allowed in strict mode. Always throw an Error object like this: `throw new Error('msg')`.
You could also allow a function to be passed so everyone can draw there own objects as points:

```js
if(typeof pointType == 'function') {
  return pointType.call(this, cx, cy)
}
```

But thats just a quick thought